### PR TITLE
feat(release): Promote Desktop, AI Summaries, Meetings and Archiving

### PR DESCRIPTION
**Pull request by nickvergessen**
grafik

### CODE REVIEW

File: lib/Chat/Changelog/Manager.php [hunk 1]
```diff
@@ -135,6 +135,11 @@ public function getChangelogs(): array {
 			. $this->l->t('- Moderators can now ban accounts and guests to prevent them from rejoining a conversation') . "\n"
 			. $this->l->t('- Upcoming calls from linked calendar events and out-of-office replacements are now shown in conversations') . "\n"
 			. $this->l->t('- Calls can now be done in a federated way with other Talk servers (requires the High-performance backend)'),
+			$this->l->t('## New in Talk %s', ['20.1']) . "\n"
+			. $this->l->t('- Desktop client for Windows, macOS and Linux is now available: %s', ['https://nextcloud.com/talk-desktop-install']) . "\n"
```
Comment (jospoortvliet) [inline lib/Chat/Changelog/Manager.php:139]:
```suggestion
			. $this->l->t('- Introducing our desktop client for Windows, macOS and Linux: %s', ['https://nextcloud.com/talk-desktop-install']) . "\n"
```

File: lib/Chat/Changelog/Manager.php [hunk 2]
```diff
@@ -135,6 +135,11 @@ public function getChangelogs(): array {
...
+			. $this->l->t('- Summarize call recordings and chats with many unread messages (requires AI configuration in the server)') . "\n"
```
Comment (jospoortvliet) [inline lib/Chat/Changelog/Manager.php:140]:
```suggestion
			. $this->l->t('- Summarize call recordings and unread messages in chats with the Nextcloud Assistant') . "\n"
```

Comment (jospoortvliet) [reply lib/Chat/Changelog/Manager.php:140]: I think just naming the Assistant makes it clear enough you need that for it to work.

**Review by jospoortvliet — CHANGES_REQUESTED** [review]
Looking good!!!!!

File: lib/Chat/Changelog/Manager.php [hunk 3]
```diff
@@ -135,6 +135,11 @@ public function getChangelogs(): array {
 			. $this->l->t('- Moderators can now ban accounts and guests to prevent them from rejoining a conversation') . "\n"
 			. $this->l->t('- Upcoming calls from linked calendar events and out-of-office replacements are now shown in conversations') . "\n"
 			. $this->l->t('- Calls can now be done in a federated way with other Talk servers (requires the High-performance backend)'),
+			$this->l->t('## New in Talk %s', ['20.1']) . "\n"
+			. $this->l->t('- Introducing the Nextcloud Talk desktop client for Windows, macOS and Linux: %s', ['https://nextcloud.com/talk-desktop-install']) . "\n"
```
Comment (nickvergessen) [inline lib/Chat/Changelog/Manager.php:139]: URL to be confirmed by @peterwebdesign

**Review by AndyScherzinger — APPROVED** [review]

**Review by DorraJaouad — APPROVED** [review]
LGTM

**Comment by nickvergessen** [comment]
Redirect is in place (linking to the wrong anchor, but we can merge already, so translators can translate over the weekend)

**Pull request merged**